components/combobox/css/_mixin.scss

```diff
@@ -0,0 +1,82 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021-2022 Gemeente Utrecht
+ * Copyright (c) 2021-2022 Frameless B.V.
+ */
+
+@mixin utrecht-combobox {
+  max-inline-size: var(--utrecht-combobox-max-inline-size);
+  position: relative;
+}
+
+@mixin utrecht-combobox_list {
+  background-color: var(--utrecht-combobox-list-background-color);
+  box-shadow: 0 4px 6px -1px var(--utrecht-combobox-list-box-shadow),
+    0 2px 4px -2px var(--utrecht-combobox-list-box-shadow);
+  box-sizing: border-box;
+  color: var(--utrecht-combobox-list-color);
+  font-size: var(--utrecht-combobox-list-font-size);
+  list-style: none;
+  margin-block: 0;
+  max-height: var(--utrecht-combobox-list-max-height);
+  overflow: scroll;
+  overflow-x: hidden;
```

Comment:
Misschien kunnen we ook éxtra `overflow-inline` toevoegen? Nog niet goede support, maar wel handig om te hebben voor browsers die het wel supporten.